checkov/terraform/checks/resource/aws/ElasticsearchDomainHA.py

```diff
@@ -0,0 +1,38 @@
+from __future__ import annotations
+
+from typing import Any
+
+from checkov.terraform.checks.resource.base_resource_check import BaseResourceCheck
+from checkov.common.models.enums import CheckCategories, CheckResult
+
+
+class ElasticsearchDomainHA(BaseResourceCheck):
+    def __init__(self) -> None:
+        """NIST.800-53.r5 CP-10, NIST.800-53.r5 CP-6(2), NIST.800-53.r5 SC-36, NIST.800-53.r5 SC-5(2),
```

Comment:
is this line relevant? seems like some random characters

Reply:
seems like ref for NIST publications

Reply:
yeah it is the NIST ref 🙂